applications/GeoMechanicsApplication/tests/surface_interface_elements/Dirichlet_multi_stage_3_plus_3/ProjectParameters_stage1.json

```
@@ -0,0 +1,140 @@
+                "output_name": "Dirichlet_multi_stage_3_plus_3_stage1",
+                "postprocess_parameters": {
+                    "result_file_configuration": {
+                        "gidpost_flags":       {
+                            "WriteDeformedMeshFlag": "WriteUndeformed",
+                            "WriteConditionsFlag":   "WriteElementsOnly",
+                            "GiDPostMode":           "GiD_PostAscii",
+                            "MultiFileFlag":         "SingleFile"
+                        },
+                        "file_label":          "step",
+                        "output_control_type": "step",
+                        "output_interval":    1,
+                        "body_output":         true,
+                        "node_output":         true,
+                        "skin_output":         false,
+                        "plane_output":        [],
+                        "nodal_results":       ["DISPLACEMENT"],
+                        "gauss_point_results": []
+                    },
+                    "point_data_configuration":  []
+                }
+            }
+        }]
+    },
+    "processes": {
+        "constraints_process_list": [{
+            "python_module": "apply_vector_constraint_table_process",
+            "kratos_module": "KratosMultiphysics.GeoMechanicsApplication",
+            "process_name":  "ApplyVectorConstraintTableProcess",
+            "Parameters":    {
+                "model_part_name": "PorousDomain.FixedNodes",
+                "variable_name":   "DISPLACEMENT",
+                "active":          [true,true,true],
+                "is_fixed":        [true,true,true],
+                "value":           [0.0,0.0,0.0],
+                "table":           [0,0,0]
+            }
+        },{
+            "python_module": "apply_vector_constraint_table_process",
+            "kratos_module": "KratosMultiphysics.GeoMechanicsApplication",
+            "process_name":  "ApplyVectorConstraintTableProcess",
+            "Parameters":    {
+                "model_part_name": "PorousDomain.SideZX",
+                "variable_name":   "DISPLACEMENT",
+                "active":          [true,true,true],
+                "is_fixed":        [true,true,true],
+                "value":           [0.0,0.0,0.0],
+                "table":           [1,2,3]
+            }
+        },{
+            "python_module": "apply_scalar_constraint_table_process",
+            "kratos_module": "KratosMultiphysics.GeoMechanicsApplication",
+            "process_name":  "ApplyScalarConstraintTableProcess",
+            "Parameters":    {
+                "model_part_name": "PorousDomain.AllNodes",
+                "variable_name":   "WATER_PRESSURE",
+                "active":          true,
+                "is_fixed":        true,
+                "value":           0.0,
+                "table":           0
+            }
+        }],
+        "loads_process_list": []
```

Empty list can be removed:
```suggestion
        }]
```